ApplicationPattern.yaml

```diff
@@ -681,6 +695,16 @@ paths:
                           description: >
                             'Own release number
                             from [/core-model-1-4:control-construct/logical-termination-point=*-1-0-0-http-s-000/layer-protocol=0/http-server-interface-1-0:http-server-interface-pac/http-server-interface-capability/release-number]'
+                        preceding-application-name:
```

Comment:
If we include the property "minLength: 3" for this attribute in RegistryOffice/register-application service , shall we update it here as well? Kindly let me know thanks.